.github/workflows/KubernetesExternalClients.yaml

```diff
@@ -217,14 +217,26 @@ jobs:
       with:
         repository: hazelcast/hazelcast-nodejs-client
         path: KubernetesExternalClients/nodejs/clientSourceCode
-        ref: ${{ github.event.inputs.nodejs_run }}
-        
+        ref: master
+
+    - name: Install dependencies
+      run: npm install
+      working-directory: KubernetesExternalClients/nodejs/clientSourceCode
+
+    - name: Download and prepare release pack from branch ${{ github.event.inputs.nodejs_run }}
+      run: |
+        node scripts/download-release-candidate.js -b ${{ github.event.inputs.nodejs_run }} -u github_actions -p ${{ secrets.GITHUB_TOKEN }} -o release-candidate.zip
```

Comment:
`download-release-candidate.js` script is also creating the release package or just downloading it?
I mean if there is no package created from given branch name, will it fail?